new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java

```diff
@@ -226,112 +256,380 @@ public static void initAll(@Namespaces(6) List<String> namespaces) {
-    Map<String, Integer> clusterNameMsPortMap = new HashMap<>();
+    clusterNameMsPortMap = new HashMap<>();
     clusterNameMsPortMap.put(clusterName, managedServerPort);
-    ingressHostList =
-        createIngressForDomainAndVerify(domain1Uid, domain1Namespace, clusterNameMsPortMap);
+    ingressHost1List =
+        createIngressForDomainAndVerify(domain1Uid, domain1Namespace,clusterNameMsPortMap, false);
+
+    exporterUrl = String.format("http://%s:%s/wls-exporter/",K8S_NODEPORT_HOST,nodeportshttp1);
+
+    //create pv and pvc for monitoring
+    HashMap<String, String> labels = new HashMap<>();
+    labels.put("app", "monitoring");
+    labels.put("weblogic.domainUid", domain1Uid);
+    assertDoesNotThrow(() -> createPvAndPvc("prometheus", monitoringNS, labels));
+    assertDoesNotThrow(() -> createPvAndPvc("alertmanager",monitoringNS, labels));
+    assertDoesNotThrow(() -> createPvAndPvc("grafana", monitoringNS, labels));
 
   }
 
   /**
-   * Test covers the following use cases.
+   * Test covers end to end sample, provided in the Monitoring Exporter github project .
    * Create Prometheus, Grafana, Webhook, Coordinator
-   * create domain Model in Image with monitoring exporter
+   * create domain in Image with monitoring exporter
    * verify access to monitoring exporter WebLogic metrics via nginx
-   * check generated by monitoring exporter WebLogic metrics via Prometheus
+   * check generated by monitoring exporter WebLogic metrics via Prometheus, Grafana
+   * fire Alert using Webhook
+   * change prometheus to add different domain to monitor
    */
   @Test
-  @DisplayName("Install Prometheus, Grafana , Webhook, Coordinator and verify WebLogic metrics")
-  public void testCheckMetrics() throws Exception {
+  @DisplayName("Test End to End example from MonitoringExporter github project.")
+  public void testEndToEndViaChart() throws Exception {
+    wdtImage = createAndVerifyDomainInImage();
+    try {
+      // create and verify one cluster wdt domain
+      logger.info("Create wdt domain and verify that it's running");
+      createAndVerifyDomain(wdtImage, domain2Uid, domain2Namespace, "Image", replicaCount);
+      ingressHost2List =
+              createIngressForDomainAndVerify(domain2Uid, domain2Namespace, clusterNameMsPortMap);
+
+      installPrometheusGrafana(PROMETHEUS_CHART_VERSION, GRAFANA_CHART_VERSION,
+              domain2Namespace,
+              domain2Uid);
+
+      installWebhook();
+      installCoordinator(domain2Namespace);
+
+      //verify access to Monitoring Exporter
+      verifyMonExpAppAccessThroughNginx(ingressHost2List.get(0), managedServersCount);
+      //verify metrics via prometheus
+      String testappPrometheusSearchKey =
+              "wls_servlet_invocation_total_count%7Bapp%3D%22test-webapp%22%7D%5B15s%5D";
+      checkMetricsViaPrometheus(testappPrometheusSearchKey, "test-webapp");
+      //fire alert by scaling down
+      fireAlert();
+      //switch to monitor another domain
```

Comment:
The above comments can be logged as messages, easier to debug when test fails.

Reply:
done